schedule/yam/agama_gnome_unattended.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: agama_gnome_unattended
+description: >
+  Prepare unattended installation with agama gnome pattern.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/agama_auto
+  - installation/grub_test
+  - yam/agama/switch_to_product
```

Comment:
this wasn't needed anymore afir.

Reply:
OK, I'd run  with the 3 modules `system_prepare hostname force_scheduled_tasks` for x86_64 and aarch64.

Reply:
yes. we need switch_to_product, we need to reset_consoles because we've select root console in import_agama_profile. [this one is w/o switch_to_product](https://openqa.suse.de/tests/17501835#step/system_prepare/3)

Reply:
ok, then, do not schedule those modules if they don't help.
I can see that we end up in the tty6 anyway in both cases. could be missing a needle?